### DISCUSSION TITLE
Referring to Facebook ane, IOS package can't generate dysm file

### DISCUSSION THREAD

**Discussion by danuomi** [airsdk/Adobe-Runtime-Support#748] [Q&A]
Hello, our IOS project uses ane. They are Facebook core, Facebook login and Facebook share. They are packaged with Airsdk33.1.1.406. There is a problem with the size of the generated dysm file. Can the airsdk packaging command output logs? We want to know what's blocking the generation of dysm. thank you

**Comment by danuomi** [comment]
https://github.com/Gamua/Adobe-Runtime-Support/issues/521#


We also have this problem. Re sign IPA with a new signature script

**Comment by CoolFishGames** [comment]
We got the same problem also, dSYM generate failed.

**Comment by ajwfrost** [comment]
Have you tried using the updated ADT package that has the fixes in for #521? This will be pushed into our next SDK release due in the next few days.. if there's still an issue with the dSYM generation then we'll take a closer look, the only things that should have changed recently are around the code-signing so potentially this is related..

**Reply by danuomi** [reply]
We used a fix for adt.jar .Can I see the log that generated dSYM.I want to fix the problem First.Can I see the log of packing dsym, if possible, which directory he is in. My packaging environment is MacOS 10.14.6

**Reply by danuomi** [reply]
![1616554613(1)](https://user-images.githubusercontent.com/81250511/112247679-b3d19180-8c8f-11eb-885e-1065b5e02cac.jpg)

**Comment by marchbold** [comment]
I've used the `adt.jar` from #521 and seems to be generating a dSYM correctly for me.
